specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -2491,6 +2617,111 @@
                     "$ref": "#/definitions/ExternalSecuritySolutionProperties"
                 }
             ]
+        },
+        "AllowedConnectionsList": {
+            "type": "object",
+            "description": "List of allowed connections",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "readOnly": true,
+                    "items": {
+                        "$ref": "#/definitions/AllowedConnectionsResource"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "AllowedConnectionsResource": {
+            "type": "object",
+            "description": "Describes an allowed connections resource",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "readOnly": true,
+                    "$ref": "#/definitions/AllowedConnectionsResourceProperties"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                },
+                {
+                    "$ref": "#/definitions/Location"
+                }
+            ]
+        },
+        "AllowedConnectionsResourceProperties": {
+            "type": "object",
+            "description": "Describes properties of an allowed connections resource",
+            "properties": {
+                "calculatedDateTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "readOnly": true,
+                    "description": "The UTC time on which the allowed connections resource was calculated"
+                },
+                "connectableResources": {
+                    "type": "array",
+                    "readOnly": true,
+                    "description": "List of connectable resources",
+                    "items": {
+                        "$ref": "#/definitions/ConnectableResource"
+                    }
+                }
+            }
+        },
+        "ConnectableResource": {

```

Comment:
@yanivn3

Reply:
Chen, the Rest is already in public and approved by ARM (since last March). We will improve documentation but changing fields is a breaking change that I prefer to avoid.

Reply:
@yanivn3  ok, so can you please improve the documentation as suggested?
please keep that comment in mind when we will create our new API version